src/lambdas/initiateAthenaQuery/handler.test.ts

```diff
@@ -42,152 +27,8 @@ jest.mock('../../sharedServices/sns/publishToSNS', () => ({
-    mockConfirmAthenaTable.mockResolvedValue({
-      tableAvailable: false,
-      message: 'test error message'
-    })
-    const testZendeskId = testAthenaQueryEvent.Records[0].body
-    await expect(
-      handler(testAthenaQueryEvent, mockLambdaContext)
-    ).rejects.toThrow('test error message')
-    expect(mockConfirmAthenaTable).toHaveBeenCalled()
-    expect(mockUpdateZendeskTicket).toHaveBeenCalledWith(
-      testZendeskId,
-      'test error message',
-      'closed'
-    )
-  })
-
-  it('updates zendesk and throws an error if no query sql is generated', async () => {
-    mockConfirmAthenaTable.mockResolvedValue({
-      tableAvailable: true,
-      message: 'test message'
-    })
-    mockGetDatabaseEntryByZendeskId.mockResolvedValue({
-      requestInfo: noIdTestDataRequest
-    })
-    mockCreateQuerySql.mockReturnValue({
-      sqlGenerated: false,
-      error: 'sql error message'
-    })
-    const testZendeskId = testAthenaQueryEvent.Records[0].body
-
-    await expect(
-      handler(testAthenaQueryEvent, mockLambdaContext)
-    ).rejects.toThrow('sql error message')
-    expect(mockConfirmAthenaTable).toHaveBeenCalled()
-    expect(mockGetDatabaseEntryByZendeskId).toHaveBeenCalledWith(testZendeskId)
-    expect(mockCreateQuerySql).toHaveBeenCalledWith(noIdTestDataRequest)
-    expect(mockUpdateZendeskTicket).toHaveBeenCalledWith(
-      testZendeskId,
-      'sql error message',
-      'closed'
-    )
-    expect(mockUpdateQueryByZendeskId).not.toHaveBeenCalled()
-  })
-
-  it('updates zendesk and throws an error if it is unable to initiate an athena query', async () => {
-    mockConfirmAthenaTable.mockResolvedValue({
-      tableAvailable: true,
-      message: 'test message'
-    })
-    mockGetDatabaseEntryByZendeskId.mockResolvedValue({
-      requestInfo: dataPathsTestDataRequest
-    })
-    mockCreateQuerySql.mockReturnValue({
-      sqlGenerated: true,
-      sql: 'test sql string',
-      queryParameters: ['123']
-    })
-    mockStartQueryExecution.mockResolvedValue({
-      queryExecuted: false,
-      error: 'test athena error'
-    })
-
-    const testZendeskId = testAthenaQueryEvent.Records[0].body
-
-    await expect(
-      handler(testAthenaQueryEvent, mockLambdaContext)
-    ).rejects.toThrow('test athena error')
-    expect(mockConfirmAthenaTable).toHaveBeenCalled()
-    expect(mockGetDatabaseEntryByZendeskId).toHaveBeenCalledWith(testZendeskId)
-    expect(mockCreateQuerySql).toHaveBeenCalledWith(dataPathsTestDataRequest)
-    expect(mockStartQueryExecution).toHaveBeenCalledWith({
-      sqlGenerated: true,
-      sql: 'test sql string',
-      queryParameters: ['123']
-    })
-    expect(mockUpdateZendeskTicket).toHaveBeenCalledWith(
-      testZendeskId,
-      'test athena error',
-      'closed'
-    )
-    expect(mockUpdateQueryByZendeskId).not.toHaveBeenCalled()
-  })
-})
-
 describe('tests related to running manual queries', () => {
```

Comment:
You need a test for the "normal" scenario, where now you'll be asserting that the parameters from the handler are passed through correctly. You also need to reinstate some of the tests that are to do with the event body being malformed.

Reply:
done